.eslintrc-server-test

```diff
@@ -0,0 +1,16 @@
+---
+extends:
+  - "defaults/configurations/walmart/es5-test"
+  - "defaults/configurations/walmart/es5-node"
```

Comment:
`es5-test`'s overrides are all duplicated here due to the issue with extending two configurations (the issue that spawned the new `es6-react-test` config). Maybe just remove `es5-test`? At present we need to assume that extending multiple configs like this causes unexpected outcomes.